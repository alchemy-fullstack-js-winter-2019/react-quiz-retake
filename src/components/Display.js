import React from 'react';
import PropTypes from 'prop-types';
import Title from './Title';

function Display({ titles }) {
  const listOfTitles = titles.map(title => {
    return (
      <li key={title.name}>
        <Title title={title} value={title} />
      </li>
    );
  });
  return (
    <ul>
      {listOfTitles}
    </ul>
  );
}
Display.propTypes = {
  titles: PropTypes.array.isRequired
};

export default Display;

