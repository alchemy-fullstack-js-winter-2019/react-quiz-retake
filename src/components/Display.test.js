import React from 'react';
import renderer from 'react-test-renderer';
import Display from './Display';

describe('Display', () => {
  it('matches a snapshot', () => {
    const display = {
      title: 'fluffy',
      subtitle: 'alive',
    };
    const tree = renderer.create(
      <Display display={display} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
