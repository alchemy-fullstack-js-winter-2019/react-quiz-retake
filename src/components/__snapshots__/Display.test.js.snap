// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches a snapshot 1`] = `
Array [
  <h1>
     
     
  </h1>,
  <h2>
     
     
  </h2>,
]
`;
